user_management/api/serializers.py

```diff
@@ -164,6 +168,22 @@ class Meta:
         read_only_fields = ('email', 'date_joined')
 
 
+class ResendConfirmationEmailSerializer(EmailSerializerBase):
+    def validate_email(self, attrs, source):
+        """Check if user exists and email_verification_required."""
+        email = attrs[source]
+        try:
+            self.user = User.objects.get_by_natural_key(email)
+        except User.DoesNotExist:
+            msg = _('User does not exists.')
+            raise serializers.ValidationError(msg)
+
+        if not self.user.email_verification_required:
```

Comment:
I'm using `user` back to the view https://github.com/incuna/django-user-management/pull/97/files#diff-6ad02d5ef719853505cf383ec7487a84R240 as I'm using it to send email